src/main/kotlin/no/nav/helse/behandling/SykepengeVedtak.kt

```diff
@@ -1,18 +1,18 @@
 package no.nav.helse.behandling
 
-import no.nav.helse.behandling.søknad.Sykepengesøknad
 import no.nav.helse.sykepenger.beregning.Beregningsresultat
 import no.nav.nare.core.evaluations.Evaluering
 import java.math.BigDecimal
 import java.time.LocalDate
 import java.util.UUID
 
 data class SykepengeVedtak(
-        val originalSøknad: Sykepengesøknad,
+        val sakskompleks: Sakskompleks,
         val faktagrunnlag: Faktagrunnlag,
         val avklarteVerdier: AvklarteVerdier,
         val vilkårsprøving: Evaluering,
         val beregning: Beregningsresultat,
+        val beregningFraInntektsmelding: Beregningsresultat,
```

Comment:
hvorfor lager vi egen beregning for inntektsmelding?

Reply:
Morten ville ha med både den opprinnelige beregningen (uten inntektsmelding) og beregningen med inntektsmelding. Jeg fant ikke en bedre måte å gjøre dette på, enn dette. Men jeg tar gjerne innspill! Det kunne jo vært en liste med flere beregninger i?